### ISSUE TITLE
.NET 4.5

### ISSUE THREAD

**Issue by doczoidberg** [df3xc/pmDotNet#1]
did you get it to run with 4.5? I had no success for now

**Comment by sveni2211** [comment]
Yes, it's running. But if you use Visual Studio higher than VS2008 you got exceptions on every call to the libraray. Because of the Calling Convention is wrong. VS2008 ignored this.

Just change every line of "[DllImport("PMDLL.dll")]" to [DllImport("PMDLL.dll", CallingConvention = CallingConvention.Cdecl)] and it will run.
